packages/gestalt/src/Tabs.css

```diff
@@ -3,6 +3,10 @@
   user-select: none;
 }
 
+.TabsWrap {
+  composes: flexWrap from "./Layout.css";
```

Comment:
Since this class only composes from another CSS file, can we not duplicate the class and instead import the `Layout.css` file into the `Tabs.js` component and then use it directly there